common/reviews/api/streaming-image-volume-loader.api.md

```diff
@@ -907,6 +1021,8 @@ interface IViewport {
     id: string;
     isDisabled: boolean;
     options: ViewportInputOptions;
+    removeActor(actorUID: string): void;

```

Comment:
Please just leave 1 call, removeActor(...actors: string) - that way you just apply it to one or more items.